Price Calculator/DiscountInformation.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Price_Calculator
+{
+    public class DiscountInformation
+    {
+
+        public enum DiscountType
+        {
+            AfterTax,
+            BeforeTax
+        }
+        public enum DiscountMethod
+        {
+            Additive,
+            Multiplicative
+        }
+        public static decimal Discount { get; set; }
+        public static decimal UPCDiscount { get; set; } = 0;  // Percentage 
+        public static int DiscountedUPC { get; set; } = 0;  // applied for this UPC only
+
+        public static decimal Cap { get; set; }
+        public static bool IsCapPercentageValue { get; set; }
```

Comment:
here same suggestion as UPC discount, extract into type and declare here property with the new type